techniques/general/G211.html

```diff
@@ -149,7 +149,30 @@ <h4>Range of inputs with few labels</h4>
 </p>
 				<p class="working-example"><a href="../../working-examples/label-in-name-general/example4.html" target="_blank">Working example of range of inputs</a></p>
 		</section>
-							
+	
+		<section class="example">
+			<h3>Parenthetical text and punctuation</h3>
+			<p>Technique G211 is not intended to complicate existing conventions for the construction of accessible form inputs. As described in the <em>Punctuation and capitalization</em> and <em>Parenthetical text</em> subsections of the Understanding document, information that would not normally be spoken when using speech recognition to navigate to controls does not need to be in the accessible name.  Where there are established ways of ensuring information and relationships conveyed visually are present programmatically, it is acceptable to leave parenthetical information and punctuation out of the accessible name.</p>
+			<p>The following snippet demonstrates possible techniques mentioned in the Understanding document. Since the required field is programmatically indicated, and the input restrictions on the date are surfaced through <code>aria-describedby</code>, the asterisk and parenthetical information is not required in the accessible name.</p>
+		
+
+<code>	
+&lt;label for="name"&gt;Name&lt;/label&gt; *<br />
+&lt;input type="text" name="name" id="name" required&gt;<br />
+&lt;label for="birth"&gt;Birthdate&lt;/label&gt; <span id="mask">(YYYY-MM-DD></span><br />
```

Comment:
does the span for the mask also need to have its `<` and `>` changed to `&lt;` and `&gt;`??

Reply:
yes, it should

Reply:
i thought I already correct that. Is it still showing up for you?